### DISCUSSION TITLE
Error : Unsupported pyinstaller version or not a pyinstaller archive

### DISCUSSION THREAD

**Discussion by andreyrisukhin** [extremecoders-re/pyinstxtractor#12] [Q&A]
When using this script, I am getting the message `Error : Unsupported pyinstaller version or not a pyinstaller archive`, even though I have installed PyInstaller. Is there a specific place PyInstaller should be, or a path variable I should modify? Thank you!

EDIT: I ran one of the test files, and it worked. This makes me wonder whether my .exe file was not compiled in PyInstaller. Is this what "...not a pyinstaller archive" means?

**Comment by extremecoders-re** [comment]
Pyinstxtractor doesn't require PyInstaller to be installed or present on the system path. It's completely standalone and requires just Python.
However, in case the binary is signed and/or modified pyinstxtrator won't be able to work. 

These may be applicable -> https://github.com/extremecoders-re/pyinstxtractor/issues/9 and https://github.com/extremecoders-re/pyinstxtractor/wiki/Frequently-Asked-Questions#how-to-extract-digitally-signed-exes
